lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/rx/impl/RxThreadPoolStage.java

```diff
@@ -168,7 +168,7 @@ public void run() {
   /**
    * Closes the stage.
    *
-   * @return Exception
```

Comment:
Why in ThreadPoolStage.java you delete this clause altogether, and here convert it to throws?
I don't think it makes sense to annotate throwing Exception, since it's the most generic thing that can happen